src/OmnichannelChatSDK.ts

```diff
@@ -913,7 +913,8 @@ class OmnichannelChatSDK {
                     RequestId: this.requestId,
                     ChatId: this.chatToken.chatId as string,
                 };
-                exceptionThrowers.throwMessagingClientInitializationFailure(error, this.scenarioMarker, TelemetryEvent.StartChat, telemetryData);
+                // Log AMS failure but do not throw to avoid blocking chat creation
```

Comment:
Every `scenarioMarker` has a starting marker (start) and ending marker (completed/failed). We removed the ending marker of `startChat`.

I suggested to use `suppressChatSDKError` with `StartChat` as marker event still, then updated the `exceptionDetails` accordingly.